src/components/SearchModal/PairList/PairList.tsx

```diff
@@ -100,18 +100,25 @@ export const PairList = ({ pairs, selectedPair, onPairSelect }: PairListProps) =
 
   return (
     <Flex overflowY="auto" flex="1">
+      {/* TODO: Replace Auto Sizer */}
+      {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
+      {/* @ts-expect-error */}
```

Comment:
@wixzi Why are we disabling ts here?

Reply:
Auto Sizer types were showing error since the return type of AutoSizer was not compatible. I will double check and update the correct reason